frontend/src/Main.js

```diff
@@ -1,13 +1,37 @@
 import React from 'react';
-import { Switch, Route } from 'react-router-dom';
+import { BrowserRouter as Switch, Route } from 'react-router-dom';
 import Home from './pages/Home';
 import ActivityPage from './pages/ActivityPage';
+import Login from './Authentication/Login';
+import Register from './Authentication/Register';
+import UserActivity from './Account/Acitivity/UserActivity'
```

Comment:
There's some spelling mistakes in the current file structure. Should be "./Account/Activity/UserActivity". The folder is also incorrectly named as well.